### PR TITLE
Redirect to request group from admin request group list

### PR DESCRIPTION
**Pull request by kevin-wangg**

### CODE REVIEW

File: client/src/components/molecules/RequestGroupTable.tsx [hunk 1]
```diff
@@ -44,7 +45,14 @@ const RequestGroupTable: FunctionComponent<Props> = (props: Props) => {
                         return g1.name < g2.name ? -1 : 1})
                     .map((requestGroup : RequestGroup) => 
                         <> 
-                            <tr key={ requestGroup._id } className="data-row">
+                            <tr 
+                                key={ requestGroup._id } 
+                                className="data-row" 
+                                onClick={() => {
+                                    console.log('clicked ' + requestGroup._id)
```
Comment (meganniu) [inline client/src/components/molecules/RequestGroupTable.tsx]: console

**Review by meganniu — APPROVED** [review]
perfect!! tysm

File: client/src/components/molecules/RequestGroupTable.tsx (hunk 1 above)
Comment (kevin-wangg) [reply client/src/components/molecules/RequestGroupTable.tsx]: removed.

**Review by chamod-gamage — APPROVED** [review]
Awesome work - LGTM!

**Pull request merged**